### ISSUE TITLE
Support for K-9 friendly sans-serif font

### ISSUE THREAD

**Issue by stnor** [mathjax/MathJax#2962]
I am trying to migrate our code to 4.0 and got things running fine, but I would need support for a child friendly sans-serif font (see mathjax/MathJax-src#172)

I don't think there are any in alpha1? Any plans on providing one soonish?

**Comment by dpvc** [comment]
The `mathjax-fira` font is a sans-serif font, so you might try that and see if it works for you.  See the [release notes](https://github.com/mathjax/MathJax-src/releases/tag/4.0.0-alpha.1#fonts) for how to configure the use of this font.

**Comment by stnor** [comment]
That's awesome! Works perfectly! THANK YOU!

**Comment by lu-kas** [comment]
If I may attach my questions to this issue. It is great to have a sans-serif font, especially for websites! @dpvc , is it possible to configure the fira font globally non-italic?  Thanks!

**Comment by dpvc** [comment]
@lu-kas, you can use the configuration

``` js
MathJax = {
  output: {
    font: 'mathjax-fira'
  },
  startup: {
    ready() {
      delete MathJax._.core.MmlTree.MmlNodes.mi.MmlMi.prototype.setInheritedAttributes;
      MathJax.startup.defaultReady();
    }
  }
};
```

to force all the variables that would usually be italic to be upright.  You can still get italic letters using `\mit{x}` if you need them.

**Comment by lu-kas** [comment]
Thank you very much! I was waiting for this (sans-serif font as global setting) for a few years now...

**Comment by dpvc** [comment]
@lu-kas, I know that v4 has been a long time in coming, but it is a *major* update in a number of ways, not just the fonts, but many other features and enhancements.  It has taken some time to get it all together.

**Comment by lu-kas** [comment]
@dpvc I hope you did not get me wrong! Sorry if you did. By saying, that I was waiting for this for such a long time, I just wanted to express my pure joy of making this possible now. I kind of arranged myself with the situation (text in sans serif and math in serif) and by chance this weekend I wanted to see if other options become possible. So, I found this issue and was super excited to eventually find a (global) solution. 

I did not want to complain about the time scales at all. And, I very much appreciate the work that is done in the open source community. THANK YOU!
